scenarios/kubernetes_e2e.py

```diff
@@ -152,17 +175,19 @@ def main(args):
     PARSER = argparse.ArgumentParser()
     PARSER.add_argument(
         '--env-file', action="append", help='Job specific environment file')
+    PARSER.add_argument(
+        '--env-template', help='Job specific environment template')
```

Comment:
ha! So will --env-file=foo.env work if foo.env contains `THIS=$THAT` lines? I believe it might?

If so can you handle this with multiple env-files? If not I would consider either a) writing out the .env template into its own file and passing that into docker's --env-file list. b) maybe creating an upgrade scenario? or c) a script to generate the upgrade .env files (which remain simple).